options.js

```diff
@@ -0,0 +1,76 @@
+// Saves options to chrome.storage
+function save_options() {
+    var dateRangeFrom = document.getElementById("dateRangeFrom").value;
+    var dateRangeTo = document.getElementById("dateRangeTo").value;
```

Comment:
Hmm, can you take a screenshot/GIF showing me what you mean? I modified these calls to use short-circuiting (i.e. `stuff.value || null`), which _might_ address your concern, but I'm more unsure than not.

Reply:
Before you added short-circuiting, if I saved the options form while those two fields were empty, they got saved as `""`, and `chrome.storage.sync.get` retrieved them as empty strings, which then got passed to `getQuery`, where they were converted by the `Number()` call from `""` to `0`.

After the change, they got saved as `null` instead of `""`, but it seems that `chrome.storage.sync.get` does not use the default values for values that were saved as `null`... maybe the default values only get used for undefined options? I'm honestly not yet clear on how it the defaults work.

I changed `options.js` to make it so that if these two fields are empty, we save the default values instead of `null`. This seems to have fixed the issue.